examples/macbookfg/release-versions.py

```diff
@@ -114,9 +114,9 @@
 
 # ======================= PACKAGE VERSIONS ===================================
 
-Geant4_version =  "10.03"
+Geant4_version =  "10.03.p02"
 CLHEP_version =  "2.3.4.3" #"2.3.1.1"
-ROOT_version = "6.08.00"
+ROOT_version = "HEAD" #"6.08.00"
```

Comment:
I don't think this will work,  since [this](https://github.com/iLCSoft/iLCInstall/blob/master/ilcsoft/root.py#L38) is not supported anymore. But I seed you updated to XCode9.

Reply:
yes, I know. Have some uncommitted code that uses GitHub - but there they have changed the versioning scheme. Not sure how to best deal with this - guess in the long run, we'd like to get ROOT from GitHub...